fetch.bs

```diff
@@ -2697,15 +2700,86 @@ functionality.
 <a for=/>fetch controller</a> or null).
 
+<p>A <dfn export>deferred fetch record</dfn> is a <a for=/>struct</a> used to maintain state needed
+to invoke a fetch at a later time, e.g., when a <code>Document</code> object is unloaded or becomes
+not <a for=Document>fully active</a>. It has the following <a for=struct>items</a>:
+
+<dl>
+ <dt><dfn export for="deferred fetch record">request</dfn>
+ <dd>A <a for=/>request</a>.
+
+ <dt><dfn export for="deferred fetch record">inactivity delay</dfn> (default null)
+ <dd>Null or a <a>duration</a>.
+
+ <dt><dfn export for="deferred fetch record">invoke state</dfn> (default "<code>deferred</code>")
+ <dd>
+  <p>"<code>deferred</code>", "<code>scheduled</code>", "<code>terminated</code>",
+  "<code>aborted</code>", or "<code>activated</code>".
+
+  <p class=note>This value can be modified <a>in parallel</a>. There could be a race condition where
+  the <code>Document</code> object's <a for=/>event loop</a> might change it to
+  "<code>deferred</code>" at the same time that it is changed to "<code>activated</code>". UAs can
+  mitigate this race condition in an <a>implementation-defined</a> manner.
+</dl>
+
+
 <hr>
 
-<p>When a <a for=fetch>fetch group</a> is
-<dfn export for="fetch group" id=concept-fetch-group-terminate>terminated</dfn>, for each associated
-<a for="fetch group">fetch record</a> whose <a for="fetch group">fetch record</a>'s
-<a for="fetch record">controller</a> is non-null, and whose <a for="fetch record">request</a>'s
-<a>done flag</a> is unset or <a for=request>keepalive</a> is false,
-<a for="fetch controller">terminate</a> the <a for="fetch group">fetch record</a>'s
-<a for="fetch record">controller</a>.
+<p>When a <a for=fetch>fetch group</a> <var>fetchGroup</var> is
+<dfn export for="fetch group" id=concept-fetch-group-terminate>terminated</dfn>:
+
+<ol>
+ <li><p><a for=list>For each</a> <a for="fetch group">fetch record</a> <var>record</var> of
+ <var>fetchGroup</var>'s <a for="fetch group">fetch records</a>, if <var>record</var>'s
+ <a for="fetch record">controller</a> is non-null and <var>record</var>'s
+ <a for="fetch record">request</a>'s <a>done flag</a> is unset and <a for=request>keepalive</a> is
+ false, <a for="fetch controller">terminate</a> <var>record</var>'s
+ <a for="fetch record">controller</a>.
+
+ <li>
+  <p><a for=list>For each</a> <a for="fetch group">deferred fetch record</a>
+  <var>deferredRecord</var> of <var>fetchGroup</var>'s
+  <a for="fetch group">deferred fetch records</a>:
+
+  <ol>
+   <li><p>Set <var>deferredRecord</var>'s <a for="deferred fetch record">invoke state</a> to
+   "<code>terminated</code>".
+
+   <li><p><a for=/>Fetch</a> <var>deferredRecord</var>'s
+   <a for="deferred fetch record">request</a>.
+  </ol>
+ </li>
+</ol>
+
+<p>When a <a for=fetch>fetch group</a> <var>fetchGroup</var> is
+<dfn export for="fetch group" id=concept-fetch-group-deactivate>deactivated</dfn>:
```

Comment:
```
When a fetch group fetchGroup is deactivated:
```

There was a long privacy-related discussion about "whether a deferred request can be sent after navigating away and in BFCached" in https://github.com/WICG/pending-beacon/issues/3#issuecomment-1286397825, and I think the conclusion is that we need to look into the [`BackgroundSync Permission`](https://wicg.github.io/background-sync/spec/) (which is on by default in Chrome).

However, if the permission is off for the context, requests should not be sent out after user navigating away that context, as it may violate the user's expectation.

While `deactivated` covers broader than `BFCached after navigating away`, it is still relevant to this section

Should the above BackgroundSync-related behavior be specified here? It doesn't sound like an implementation-specific issue.